examples/with-angular-thirdpartyemailpassword/src/app/auth/auth.component.ts

```diff
@@ -0,0 +1,24 @@
+import { AfterViewInit, Component } from "@angular/core";
+import * as React from "react";
+
+import * as ReactDOM from "react-dom";
+
+import SuperTokensReactComponent from "./supertokens";
+
+@Component({
+    selector: "app-auth",
+    template: '<div [id]="rootId"></div>',
+})
+export class AuthComponent implements AfterViewInit {
+    title = "angularreactapp";
+
+    public rootId = "rootId";
+
+    ngAfterViewInit() {
+        this.render();
+    }
+
+    private render() {
+        ReactDOM.render(React.createElement(SuperTokensReactComponent), document.getElementById(this.rootId));
```

Comment:
How do you know that this is the right way to do this?

Reply:
@jscyo you missed this.

Reply:
Looked at a number of implementations of how people were using React components in angular and used the same method. I've added comments explaining what each lifecycle hook does